tests/test_core/test_sequence.py

```diff
@@ -2596,6 +2596,7 @@ def test_gapped_by_map_segment_iter():
     moltype = get_moltype("dna")
     m, seq = moltype.make_seq("-TCC--AG").parse_out_gaps()
     g = list(seq.gapped_by_map_segment_iter(m, allow_gaps=True, recode_gaps=False))
+    assert g == ["-", "TCC", "--", "AG"]
```

Comment:
Test was missing an assertion! Not 100% sure if this is correct

Reply:
I think that's correct!